### PR TITLE
feat: add send token action

### PR DESCRIPTION
**Pull request by kurpav**
Adds an action to allow sending tokens between wallets

### CODE REVIEW

File: src/actions/sendToken.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+interface ISendTokenParams {
+  connection: Connection;
+  wallet: Wallet;
+  // token account address
+  source: PublicKey;
+  // destination wallet address
+  destination: PublicKey;
+  mint: PublicKey;
+  amount: number | u64;
```
Comment (m-sebastiyan) [inline src/actions/sendToken.ts:18]: What about having a BN here?

**Review by m-sebastiyan — DISMISSED** [review]

File: src/actions/sendToken.ts (hunk 1 above)
Comment (kurpav) [reply src/actions/sendToken.ts:18]: `u64` extends `BN`
And to be honest I don't have any idea how can we send more than 1 token from a token account with NFT

File: src/actions/sendToken.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+  try {
+    // check if the account exists
+    await Account.load(connection, destAta);
+  } catch {
+    txs.push(
+      new CreateAssociatedTokenAccount(
+        {
+          feePayer: wallet.publicKey,
+        },
```
Comment (b2kdaman) [inline src/actions/sendToken.ts:49]: Maybe let's try to move txOptions to const?

Comment (kurpav) [reply src/actions/sendToken.ts]: fixed

**Review by m-sebastiyan — APPROVED** [review]

**Review by b2kdaman — APPROVED** [review]

**Pull request merged**